config/initializers/sufia.rb

```diff
@@ -27,8 +27,8 @@
   config.resource_types = {
     "Book" => "Book",
     "Book Chapter" => "Book Chapter",
-    "Conference\/Workshop Poster" => "Conference\/Workshop Poster",
-    "Conference\/Workshop Presentation" => "Conference\/Workshop Presentation",
+    "Conference\/workshop Poster" => "Conference\/workshop Poster",
```

Comment:
It's based on https://github.com/ualbertalib/HydraNorth/issues/503. There's inconsistency between hydranorth data dictionary and the deposit form. And Conference/workshop Poster and Conference/workshop Presentation is used in the current ERA and data dictionary.